CHANGELOG.md

```diff
@@ -335,9 +335,50 @@ Example for file plugin output:
 
 - pcsampler sample code has been removed due to deprecation from v2.
 
-## ROCprofiler for rocm 6.3
+## ROCProfiler for ROCm 6.3
```

Comment:
@bgopesh Please add ROCProfiler version number

Reply:
not needed for v2